cupy/random/_generator_api.pyx

```diff
@@ -427,6 +430,53 @@ class Generator:
                      (ngood_ptr, nbad_ptr, nsample_ptr))
         return y
 
+    def logseries(self, p, size=None):
+        """Log series distribution.
+
+        Returns an array of samples drawn from the log series distribution.
+            Its probability mass function is defined as
+
+        .. math::
+           f(x) = \\frac{-p^x}{x\\ln(1-p)}.
+
+        Args:
+            p (float or cupy.ndarray of floats): Parameter of the log series
+                distribution. Must be in the range (0, 1).
+            size (int or tuple of ints, optional): The shape of the output
+                array. If `None`(default), a single value is returned if ``p``
+                is scalar. Otherwise, ``p.size`` samples are drawn.
```

Comment:
Seems sphinx grammar requires a space after the quotes.

```suggestion
                array. If ``None`` (default), a single value is returned if
                 ``p`` is scalar. Otherwise, ``p.size`` samples are drawn.
```